lib/importers/licence_transaction/tagging_csv_validator.rb

```diff
@@ -0,0 +1,46 @@
+module Importers
+  module LicenceTransaction
+    class TaggingCsvValidator
+      attr_reader :licences_tagging
+
+      def initialize(licences_tagging)
+        @licences_tagging = licences_tagging
+      end
+
+      def valid?
+        tagging_validation_errors.empty?
+      end
+
+      def errors
+        tagging_validation_errors.each { |e| puts "#{e}\n\n" }
```

Comment:
If there are errors, I'd append a message about how to rectify the errors, e.g. "Please make sure you update the licence-transaction schema file first" and then put the link to the instructions.